Makefile

```diff
@@ -156,11 +172,15 @@ test-e2e-plain:
 	TEST_E2E_VERIFY_TEKTONOBJECTS=${TEST_E2E_VERIFY_TEKTONOBJECTS} \
 	ginkgo ${TEST_E2E_FLAGS} test/e2e
 
+crds-and-resources:
+	-hack/crds-and-resources.sh uninstall
+	@hack/crds-and-resources.sh install
+
 crds:
-	-hack/crd.sh uninstall
-	@hack/crd.sh install
+	-hack/crds.sh uninstall
+	@hack/crds.sh install
```

Comment:
Note that I've proposed conflicting changes with #385.

Reply:
ok